Makefile

```diff
@@ -25,7 +25,7 @@ DID_ELEMENT_SIDETREE_REQUEST_URL ?= https://element-did.com/api/v1/sidetree/requ
 SANDBOX_CLI_IMAGE_NAME       ?= trustbloc/sandbox-cli
 
 # TrustBloc core k8s deployment scripts https://github.com/trustbloc/k8s
-TRUSTBLOC_CORE_K8S_COMMIT=d3503d84690a80c8334f4a95c9ded539b06170af
+TRUSTBLOC_CORE_K8S_COMMIT=5fc2da82579fe5d905661aad9f958c422564b08b
```

Comment:
@soluchok The related PR is merged in trustbloc/k8s. Can you please update the commit version - https://github.com/trustbloc/k8s/commit/77c7b26a28d878431a0aef86c93b16a4a97bc9e8 ?